ENTITY/CreateModelSwap.md

```diff
@@ -20,5 +20,5 @@ Network players do not see changes done with this.
 * **radius**: 
 * **originalModel**: 
 * **newModel**: 
-* **p6**: 
+* **bSurviveMapReload**:
```

Comment:
Add some parameter description if possible.

Reply:
done to the best of my abilities